### PR TITLE
I add Ansible playbook for Elasticsearch

### LINKED ISSUE #18636: Add Ansible playbook to Integrations
We have https://github.com/elastic/ansible-elasticsearch, but it's not listed in https://www.elastic.co/guide/en/elasticsearch/plugins/master/integrations.html#_supported_by_elasticsearch_2, although Puppet and Chef are.

It probably should be.

### PR DESCRIPTION
**Pull request by refactormyself**
I add Ansible playbook for Elasticsearch under Deployment - Supported by Elasticsearch

Closes #18636

### CODE REVIEW

**Comment by refactormyself** [comment]
This PR Closes Issue  #18636

File: docs/plugins/integrations.asciidoc [hunk 1]
```diff
@@ -82,6 +82,10 @@ releases 2.0 and later do not support rivers.
 [float]
 ==== Supported by Elasticsearch:
 
+* https://github.com/elastic/ansible-elasticsearch[Ansible playbook for Elasticsearch]:
+  Ansible role for 6.x/5.x Elasticsearch. Currently this works on Debian and RedHat based linux systems. 
+  It has been tested Ubuntu 14.04/16.04, Debian 8, Centos 7
```
Comment (Crazybus) [inline docs/plugins/integrations.asciidoc]: Should be "it has been tested on". Maybe something like "Tested with the latest version of 5.x and 6.x on Ubuntu 14.04/16.04, Debian 8, Centos 7"

**Review by Crazybus — CHANGES_REQUESTED** [review]

**Comment by Crazybus** [comment]
@jasontedor Left a small comment about the wording. 

But can also confirm that this is accurate and elastic/ansible-elasticsearch is an officially supported ansible playbook for Elasticsearch.

**Review by Crazybus — APPROVED** [review]

**Comment by nik9000** [comment]
Hey, CLAbot, check this!

**Comment by jasontedor** [comment]
@nik9000 Will you handle integrating this one?

**Comment by nik9000** [comment]
> @nik9000 Will you handle integrating this one?

On it!

**Comment by nik9000** [comment]
Merged to master and backported to 6.x and 6.2 branches. The docs changes should be live in about 30 minutes on all three branches.

**Comment by jasontedor** [comment]
Thanks a lot @refactormyself for the contribution, @Crazybus for the review, and @nik9000 for integrating.

**Pull request merged**